lib/lumberjack/server.rb

```diff
@@ -50,9 +50,31 @@ def initialize(options={})
 
       if @options[:ssl]
         # load SSL certificate
-        @ssl = OpenSSL::SSL::SSLContext.new
-        @ssl.cert = OpenSSL::X509::Certificate.new(File.read(@options[:ssl_certificate]))
-        @ssl.key = OpenSSL::PKey::RSA.new(File.read(@options[:ssl_key]),
+        ssl = OpenSSL::SSL::SSLContext.new
+        raw_cert = File.read(@options[:ssl_certificate])
```

Comment:
I suspect there's a library method for reading certificates from a file that we can reuse instead of crafting our own here. I don't know what that is right now, though.